package.json

```diff
@@ -38,5 +45,10 @@
     "postversion": "git push && git push --tags",
     "watch": "gulp build && gulp watch & npm start",
     "test": "npm run scss-lint -s && npm run scss-unused-variables -s"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "react": "^15.5.4",
+    "react-dom": "^15.5.4"
   }
```

Comment:
React and react-dom is used in components and development tools so I will move this to devDep. classnames is use only in components so shouldn't moved